### DISCUSSION TITLE
vscpl2drv-tcpipsrv : driver used curl for what purpose

### DISCUSSION THREAD

**Discussion by dinguluer** [grodansparadis/vscp#274] [Q&A] [answered]
Hi,

vscpl2drv-tcpipsrv : driver used curl for vscp.git repository folder for  what purpose ?
![image](https://github.com/grodansparadis/vscp/assets/370819/93b9501a-bf40-46f9-aed1-1c72fb19a39e)

If I am correct then mdf.c module uses curl. :--
![image](https://github.com/grodansparadis/vscp/assets/370819/ae090933-ed7d-46aa-8edf-583331c29c82)

Which curl version to use :--
![image](https://github.com/grodansparadis/vscp/assets/370819/18f81cdb-a129-48e0-8aab-be476d705c41)

I installed this any issue ? :--
sudo apt-get install libcurl4-openssl-dev

Regards,
Dinesh

**Comment by grodansparadis** [comment]
curl is only used for mdf download. But may be used for other stuff also in the future.

**Comment by grodansparadis** [comment, marked as answer]
yes use "dev"
